src/features/marketplace/components/AcceptOffer.tsx

```diff
@@ -71,7 +71,9 @@ const AcceptOfferContent: React.FC<{
 
   if (display.type === "collectibles") {
     const name = KNOWN_ITEMS[itemId];
-    hasItem = !!getChestItems(game)[name]?.gte(1);
+    hasItem =
+      !!getChestItems(game)[name]?.gte(1) ||
+      !!getBasketItems(game.inventory)[name]?.gte(1);
```

Comment:
```suggestion
     !!getBasketItems(game.inventory)[name]?.gte(offer.quantity);
```

I think we need to remove the resources check below and have this check handle resources as well:

```ts
  if (display.type === "resources") {
    const name = KNOWN_ITEMS[itemId];
    hasItem = !!getBasketItems(game.inventory)[name]?.gte(offer.quantity);
  }
```